analyses/bsve/mmwr/minerva.json

```diff
@@ -0,0 +1 @@
+{ "analysis_name": "MMWR data import", "analysis_type": "mmwr_data_import" }
```

Comment:
This needs to be changed to `mmwr_import_data` to match up with the widget name.